docs/content/configuration/csi_ug_config_create_replication.md

```diff
@@ -0,0 +1,46 @@
+# Creating a VolumeReplication
+
+Create a VolumeReplication yaml file to replicate a specific PersistentVolumeClaim (PVC).
+
+VolumeReplicationClass needs to be present before a VolumeSnapshot can be created. For more information, see [Creating a VolumeReplicationClass](csi_ug_config_create_vol_replicationclass.md).
+
+**Note:** Replication is also known as mirroring in many storage systems.
+
+When replicating a volume, be sure to follow all of the replication configurations, found in [Compatibility and requirements](../installation/csi_ug_requirements.md) before snapshot creation.
+
+1.  Replicate a specific  PersistentVolumeClaim (PVC) using the demo-replication.yaml.
+
+    For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
+
+    ```
+    apiVersion: replication.storage.openshift.io/v1alpha1
+    kind: VolumeReplication
+    metadata:
+      name: volumereplication-sample
+      namespace: default
+    spec:
+      volumeReplicationClass: volumereplicationclass-sample
+      replicationState: primary
+      dataSource:
+        kind: PersistentVolumeClaim
+        name: myPersistentVolumeClaim # should be in same namespace as VolumeReplication
+    ```
+
+2.  After the YAML file is created, apply it by using the `kubectl apply -f` command.
+
+    ```
+    kubectl apply -f <filename>.yaml
+    ```
+
+3.  Verify that the VolumeReplication file was created.
+
+    Run the `kubectl describe volumereplication` command.
+
+    See the **Status** section of the output for the following:
+
+    -   **Bound Volume Snapshot Content Name:** Indicates the volume is bound to the specified VolumeSnapshotContent.
+    -   **Creation Time:** Indicates when the snapshot was created.
+    -   **Ready to Use:** Indicates the volume snapshot is ready to use.
+    -   **Restore Size:** Indicates the minimum volume size required when restoring (provisioning) a volume from this snapshot.
```

Comment:
I don't think these are relevant for replication.
@lisyukanna knows the relevant values

Reply:
@oriyarde  right. these were just kept in from the snapshot file as a holder place, as I was requested to do . @lisyukanna  Please provide with any updates that should be here. thanks

Reply:
@rivkap we have 3 states for volumereplication:
- status.state: Primary
- status.state: Secondary
- status.state: Unknown

Reply:
thanks @lisyukanna .
I've updated the section. please review and approve.

Also, what happens if it's unknown? Is what I've written sufficient for the customers?
also - according to this [usage](https://github.com/csi-addons/volume-replication-operator/tree/v0.1.0#usage) section it's called `replicationState`. is this what we have in our output as well - if so, I"ll update the text here?

Reply:
AFAIK it could but should not be "unknown" (just as a snapshot status "ready to use" could but should not be "False"), so I think it's sufficient.
`replicationState` is a [separate field](https://github.com/IBM/ibm-block-csi-driver/blob/6a88b5caf5aef1471c5a742b5a494a204911079d/deploy/kubernetes/examples/demo-volumereplication.yaml#L8), and it is more of an input rather than an output - this is why I suggested adding a link to said usage section

Reply:
DONE  @lisyukanna  @oriyarde  please approve and resolve

Reply:
as I wrote, IMO:
`Indicates that the driver does not recognize the replicated state` should be:
`Indicates that the driver does not recognize the replication state` (because the state is not what is being replicated)

in addition, IMO:
`See the status.state section to see which of the following states the replicated volume is part of` should be:
`See the status.state section to see which of the following states the replication is in` or:
`See the status.state section to see in which of the following states the replication is`

Reply:
also:
`Volume Replicator Operator` should be:
`Volume Replication Operator`

Reply:
and:
`For information about changing the replication status` should be:
`For information about changing the replication state`

Reply:
DONE

Reply:
LGTM